src/model/test/Space_GTest.cpp

```diff
@@ -2235,7 +2235,7 @@ TEST_F(ModelFixture, RemoveSpikesAndOverlaps_TZ46_TZ47) {
     nSurfaces += space.surfaces().size();
   }
 
-  ASSERT_EQ(nSurfaces, 16);
+  EXPECT_EQ(nSurfaces, 16);
```

Comment:
From now on, lots of changing ASSERT_XXX to EXPECT_XXX. ASSERT is helpful when it would throw after it. eg `ASSERT_EQ(1, vector.size(); vector[0]....`. Otherwise EXPECT_ is better because it lets your test continue to report additional fail/success.